angularSTEP/src/app/home-page/home-page.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+
+//import { CardsComponent } from './cards.component';
+
+@Component({
+  selector: 'app-home-page',
+  templateUrl: './home-page.component.html',
+  styleUrls: ['./home-page.component.scss']
```

Comment:
There is already a home component? Is there any reason for making another one?

Reply:
I would put this functionality on the original home component. Also I am not just what this component is doing?